Dockerfile.ubi

```diff
@@ -16,9 +16,24 @@ FROM registry.fedoraproject.org/fedora-minimal:latest AS build
 USER root
 WORKDIR /work
 
-RUN microdnf install -y make golang-bin libseccomp-static glibc-static
+RUN microdnf install -y \
+        bzip2 \
+        curl-devel \
+        g++ \
+        glibc-static \
+        golang-bin \
+        libarchive-devel \
+        libmicrohttpd-devel \
+        libseccomp-static \
+        m4 \
+        make \
+        sqlite-devel \
+        tar \
+        zlib-static
 
 ADD . /work
+RUN hack/install-libelf.sh
```

Comment:
@jhrozek Fedora does not ship libelf static any more since a couple of releases. I think we cannot avoid building it on our own here, right?

Reply:
That's right. I guess we could ask for it, but there is no guarantee the request would be granted. Moreover, thinking about SPO on OCP and therefore on RHCOS, the selection of build libraries would be even more restricted there. I'm also wondering about multi-architecture support. Not to spill too many internal things, but I've recently worked on porting some other operators to ppc64 and Z -- if the bpf support is not fully supported on those architectures, I was wondering that we would still either use another method there or just no recording. Either of these would mean disabling the built-in bpf recorder.